README copy.md

```diff
@@ -0,0 +1 @@
+# VTube
```

Comment:
Please add a descriptive readme file

Reply:
- [ ] For a better communication, I think it would be better to make suggestions and give an explanation (why) for each requested change instead of giving orders, :joy: :joy: 
    **eg:** "_I think it would be to add a descriptive README file with all the main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) to make your README more informative and professional_" sounds nicer than "_Please add a descriptive readme file,_" doesn't it? :+1: :+1:

Reply:
As highlighted by the previous reviewer kindly add a reason why you requested the readme file. As referenced in the lesson[How to  give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) "it might be obvious for you but not for someone else."